index.html

```diff
@@ -9,6 +9,10 @@
     integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA=="
     crossorigin="anonymous" />
   <link rel="stylesheet" href="main.css">
+  <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
   <title>Document</title>
 </head>
 
```

Comment:
![Screenshot from 2023-06-01 10-52-15](https://github.com/maths29/Capstone-project-1/assets/83042742/9b590dbf-b6f3-4084-b706-4d1a2dd38c68)
- [ ] As mentioned by the previous reviewer and to match the template design, kindly ensure that you improve the font size of the main heading design to match the required one as right they look so different.

| Template Design | Your Design |
| ----------------------- | --------------------- |
| ![Screenshot from 2023-06-01 10-57-56](https://github.com/maths29/Capstone-project-1/assets/83042742/7472cfec-92fc-4ef9-bc79-cb673a465160) | ![Screenshot from 2023-06-01 10-55-32](https://github.com/maths29/Capstone-project-1/assets/83042742/d070b659-43d4-494d-b27d-3cb28703f7af) |